app/models/miq_provision/naming.rb

```diff
@@ -1,31 +1,20 @@
 module MiqProvision::Naming
   extend ActiveSupport::Concern
 
-  NAME_VIA_AUTOMATE = true
   NAME_SEQUENCE_REGEX = /\$n\{(\d+)\}/
   SOURCE_IDENTIFIER = "provisioning"  # a unique name for the source column in custom_attributes table
 
   module ClassMethods
     def get_next_vm_name(prov_obj, determine_index = true)
       unresolved_vm_name = nil
```

Comment:
I don't think you need this assignment anymore either, right?

Reply:
Correct, I was initially limiting this to just the relevant changes as I am working on another refactoring of this code to support some new functionality.

It was hard to ignore this at the end of the method, but I'll add them if you prefer:
```
vm_name = get_vm_full_name(unresolved_vm_name, prov_obj, determine_index)
vm_name
```

Reply:
Heh, gotcha. I don't mind some cleanup in the same PR (in a separate commit!), but you can do whatever you wish!